### DISCUSSION TITLE
Nginx Proxy Manager with open-appsec

### DISCUSSION THREAD

**Discussion by vdias** [tteck/Proxmox#2935] [Ideas]
Could be a good idea to get a LXC with NPM and open-appsec
[Step-by-Step](
https://docs.openappsec.io/integrations/nginx-proxy-manager-integration/install-nginx-proxy-manager-with-open-appsec-managed-from-npm-webui)

**Comment by Hotte512** [comment]
Yeah, I did not manage to install openappsec inside the LXC NginxProxyManager :-(
